Makefile

```diff
@@ -89,6 +89,11 @@ release: clean
 	python setup.py sdist upload -r pypi_internal
 	python setup.py bdist_wheel upload -r pypi_internal
 
+release-builds: clean-build clean-pyc
```

Comment:
Have you tried the following:
release-builds: clean-build clean-pyc dist dist-online
If that works, we don't need to define the dist build process a second time, if not, we should figure out why and fix it.